js/output/pjs/pjs-ast-transforms.js

```diff
@@ -304,10 +304,15 @@ ASTTransforms.rewriteNewExpressions = function(envName) {
                             b.Identifier("applyInstance")
                         ),
                         [
+                            // If we're not currently inside of a function,
+                            //  then prefixing identifiers is alright.
+                            // However, if we are inside of a function,
+                            //  then we shouldn't be prefixing function params.
+                            (/^Function/.test(path[path.length - 4].type) === false ?
```

Comment:
This seems unreliable.  What if you're inside a loop inside a function.  You should probably search for /^Function/ in all ancestor nodes.